hw/bsp/dwm1001-dev/syscfg.yml

```diff
@@ -62,3 +71,6 @@ syscfg.vals.BLE_CONTROLLER:
     OS_CPUTIME_FREQ: 32768
     OS_CPUTIME_TIMER_NUM: 5
     BLE_LL_RFMGMT_ENABLE_TIME: 1500
+
+syscfg.restrictions:
+    - "LIS2DH12_ONB && I2C_0_PIN_SCL == 28 && I2C_0_PIN_SDA == 29"
```

Comment:
if it was:
```yml
syscfg.restrictions:
    - "!LIS2DH12_ONB || (I2C_0 == 1 && I2C_0_PIN_SCL == 28 && I2C_0_PIN_SDA == 29)"
```
sensor could be turned on and off as syscfg value implies it could.
notice extra `I2C_0 == 1` that is also needed I guess